mkosi/qemu.py

```diff
@@ -156,6 +160,45 @@ def start_swtpm() -> Iterator[Optional[Path]]:
             proc.wait()
 
 
+@contextlib.contextmanager
+def start_virtiofsd(directory: Path) -> Iterator[Path]:
+    uid, gid = InvokingUser.uid_gid()
+
+    with tempfile.TemporaryDirectory() as state:
+        # Make sure virtiofsd is allowed to create its socket in this temporary directory.
+        os.chown(state, uid, gid)
+
+        sock = Path(state) / Path("sock")
+
+        virtiofsd = shutil.which("virtiofsd")
+        if virtiofsd is None:
+            if Path("/usr/libexec/virtiofsd").exists():
+                virtiofsd = "/usr/libexec/virtiofsd"
+            elif Path("/usr/lib/virtiofsd").exists():
+                virtiofsd = "/usr/lib/virtiofsd"
+            else:
+                die("virtiofsd must be installed to use RuntimeMounts= with mkosi qemu")
```

Comment:
Should this say `RuntimeTrees=` instead of `RuntimeMounts`?

Reply:
https://github.com/systemd/mkosi/pull/2176